main.js

```diff
@@ -0,0 +1,25 @@
+let tweeter = Twitter()
+const renderer = Renderer()
+renderer.rendererPosts(tweeter.getPosts())
+
+$("body").on("click", "#post", function() {
+    tweeter.addPost($("input").val())
+    renderer.rendererPosts(tweeter.getPosts())
+})
+
+$("body").on("click", ".delete", function() {
+    tweeter.removePost($(this).closest(".post").data().id)
+    renderer.rendererPosts(tweeter.getPosts())
+
+})
+
+$("body").on("click", ".addComment", function() {
+    tweeter.addComment($(this).closest("div").closest("div").data().id, ($(this).closest("div").find("input").val()))
```

Comment:
this line is a little bit too long - create two variables that holds what you send to the addComment function and then use them